src/scripts/towns/TownList.ts

```diff
@@ -4390,40 +4394,140 @@ const FreezingtonShop = new Shop([
+});
+
+const Archaeologist = new NPC('Archaeologist', [
+    'I\'ve been studying the Galarian form of Yamask.',
+    'It seems they do not evolve into Galarian Cofagrigus, but a completely different Pokémon instead.',
+    'For some mysterious reason, they only evolve when trained in a specific area of the wild area known as the Dusty Bowl.',
+], { image: 'assets/images/trainers/Ruin Maniac.png' });
+
+const HerosBath = new NPC('Hero’s Bath', [
+    '<i>The ancient Hero’s Bath.</i>',
+    '<i>They say this is where the two heroes came to soothe their wounds after the battle to bring down that evil presence, long ago.</i>',
+], {image: 'assets/images/npcs/other/HerosBath.png'});
+
+const CirchesterHop = new NPC('Hop', [
+    'Based on the statues the sword and shield must have actually been two Pokémon, right?',
+    'You remember the Pokémon we met in the Slumbering Weald?',
+    'Do you think...it could’ve been one of them? I mean the sword or the shield Pokémon?',
+    'Perhaps when their duty was completed, they went into some kind of sleep?',
+], {
+    image: 'assets/images/temporaryBattle/Hop.png',
+    requirement: new MultiRequirement([new TemporaryBattleRequirement('Hop7'), new QuestLineCompletedRequirement('The Darkest Day', GameConstants.AchievementOption.less)]),
+});
+
+const CirchesterSonia = new NPC('Sonia', [
+    'Hmm... Who exactly were the heroes that bathed here?',
+    'Nowadays only Pokémon really use the Hero’s Bath...',
+    'I think I’ll have to look more into the history of the Slumbering Weald.',
+], {
+    image: 'assets/images/npcs/Sonia.png',
+    requirement: new MultiRequirement([new TemporaryBattleRequirement('Hop7'), new QuestLineCompletedRequirement('The Darkest Day', GameConstants.AchievementOption.less)]),
+});
+
+const CirchesterGuitarist = new NPC('Guitarist Justin', [
+    'I caught this Snom on Steamdrift Way and we’ve become really good mates.',
+    'I thought that friendship was all Snom needed to evolve, but I kept trying to give it a Soothe Bell during the day and nothing happened!',
+    'I’m too busy playing gigs to scamper around with Snom in the evening!',
+], {image: 'assets/images/trainers/Guitarist (male).png'});
+
 const TeamYellGrunts = new NPC('Team Yell Grunts', [
-    'Our Marnie\'s Morpeko is so cute, don\'tcha think?',
-    'When it\'s hungry, though, it gets so angry it changes forms',
+    'Our Marnie’s Morpeko is so cute, don’tcha think?',
+    'When it’s hungry, though, it gets so angry it changes forms.',
     'If you want to catch it in that form, you would probably be best trying to attract it with a berry that electric Pokémon like.',
 ], {image: 'assets/images/trainers/Team Yell Grunts.png'});
+
+/* To be added when held item evos are ready.
+// TODO: Add to town near a dungeon/shop that has the Leek available.
+const FarfetchdArtist = new NPC('Artist Duncan', [
```

Comment:
Leek will be added as a drop item for Galarian Farfetch'd, so you can just finish this NPC